ui/src/org/pentaho/di/ui/core/database/dialog/XulDatabaseDialog.java

```diff
@@ -141,7 +142,7 @@ private void createDialog() {
     try {
       databaseDialogInstance = new DatabaseConnectionDialog();
       databaseDialogInstance.registerClass( EXTENDED_WIDGET_ID, EXTENDED_WIDGET_CLASSNAME );
-      container = databaseDialogInstance.getSwtInstance( parentShell ); // Attention: onload: loadConnectionData() is
+      container = databaseDialogInstance.getSwtInstance(new KettleXulLoader(), parentShell ); // Attention: onload: loadConnectionData() is
```

Comment:
@AlexanderBuloichik checkstyle issue on this line.  I haven't run the checker yet, but I know it will catch this.